mifosng-android/src/main/res/layout/fragment_savings_account_summary.xml

```diff
@@ -19,14 +19,15 @@
 
     <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
```

Comment:
Changes in this file are not related to this issue.

Reply:
@luckyman20 Actually  #1281 #1282
Both issues are solved in this pull request. There was a small issue so I squashed the commits.
Should I create 2 separate PRs for the same?

Reply:
> @luckyman20 Actually #1281 #1282
> Both issues are solved in this pull request. There was a small issue so I squashed the commits.
> Should I create 2 separate PRs for the same?

You need to make single PR for single issue.